### PR TITLE
Suggest some changes on identified typos

### PR DESCRIPTION
**Pull request by AlbaRoza**
## Subject

Per Carlos the Sailor's request I took a look at the codebase and subsequently spotted several typos.

### CODE REVIEW

**Comment by AlbaRoza** [comment]
I have read the DCO Document and I hereby sign the DCO

**Comment by jordisala1991** [comment]
Hi! Thanks for your contribution! You are the first one creating a PR outside the core team :)

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # This file is part of the Stooa codebase.
 #
-# (c) 2020 - present Runroom SL
+# (c) 2021 - present Runroom SL
```
Comment (jordisala1991) [inline .gitignore:3]: Changes on the license should be reverted since 2020 was the year we started working on Stooa.

**Comment by aerrasti** [comment]
🤩  You are our first contributor 🥇  Thank you so much for your kind cooperation 🎉  🎉

**Comment by eddsaura** [comment]
Wooooo 🙌 thanks a lot @AlbaRoza 😺

File: .gitignore (hunk 1 above)
Comment (AlbaRoza) [reply .gitignore:3]: I see. Please just ignore this suggestion, then!

File: .editorconfig [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # This file is part of the Stooa codebase.
 #
-# (c) 2020 - present Runroom SL
+# (c) 2021 - present Runroom SL
```
Comment (jordisala1991) [inline .editorconfig:3]:
```suggestion
# (c) 2020 - present Runroom SL
```

File: .env [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # This file is part of the Stooa codebase.
 #
-# (c) 2020 - present Runroom SL
+# (c) 2021 - present Runroom SL
```
Comment (jordisala1991) [inline .env:3]:
```suggestion
# (c) 2020 - present Runroom SL
```

File: .gitignore (hunk 1 above)
Comment (jordisala1991) [inline .gitignore:3]:
```suggestion
# (c) 2020 - present Runroom SL
```

**Review by jordisala1991 — APPROVED** [review]

**Review by aerrasti — APPROVED** [review]

**Review by aerrasti — APPROVED** [review]

**Pull request merged**